model-zoo-api/python_api_script.py

```diff
@@ -0,0 +1,89 @@
+# import model dictionaries
+from dict.models import modelDict
+from dict.model_versions import versionDict
+
+# download model through url
+import urllib.request
+
+import onnx
+from onnxruntime import InferenceSession
+import sys
+import os
+
+# regular expression
+import re
+
+def get_model_versions(model):
+        # call model versions dictionaries
+        if versionDict.get(model) != None:
+             print(versionDict.get(model))
+        else:
+            print("model name does not exist")
+            sys.exit()
+
+class onnx_zoo:
+    
+    def __init__(self):
+        
+        name = input("Enter model name: ")
+        get_model_versions(name)
+        model_name = input("Enter model name from options: ")
+        saved_path = input("Enter saved path: ") + "/"
```

Comment:
Wording here might need to be changed. Relevant comment included above in readme.